scripts/local-dev.sh

```diff
@@ -389,6 +384,10 @@ main() {
     echo -e "   ${YELLOW}📊 Prometheus: $PROMETHEUS_URL${NC}"
     echo -e "   ${YELLOW}🔍 TempoStack: $TEMPO_URL${NC}"
     echo -e "   ${YELLOW}🦙 LlamaStack: $LLAMA_STACK_URL${NC}"
+    echo -e "   ${YELLOW}🧩 MCP Server (health): http://localhost:$MCP_PORT/health${NC}"
```

Comment:
New lines (387-390) are already present on line 382-386. I changed them to simply use the pre-set variables instead of pointing to `localhost` again

Reply:
done